setup.py

```diff
@@ -25,15 +25,15 @@ def run(self):
     subprocess.call(['py', '-2', 'vboxapisetup.py', 'install'])
     os.chdir(old_path)
 
-setup(name='mte',
-      version='0.1.0',
-      description='Testing environment for Medusa security system',
-      url='https://github.com/Programator2/medusa-testing-environment',
-      author='Roderik Ploszek',
+setup(name='mits',
+      version='2.0.0',
+      description='Testing integration system for Medusa security system',
+      url='https://github.com/medusa-team/medusa-testing-environment',
+      author='Team Medusa',
```

Comment:
Use a person's name. Me or you, I'll let you decide. And leave/change email accordingly.

Reply:
I left there your name.